campaigns/api.py

```
@@ -0,0 +1,358 @@
+from api.pagination import CustomSizePageNumberPagination
+from base.logging import logger
+
+from .models import Campaign, CampaignDonation
+from .serializers import (
+    CampaignSerializer,
+    CampaignDonationSerializer,
+    CampaignContractConfigSerializer,
+    PaginatedCampaignsResponseSerializer,
+    PaginatedCampaignDonationsResponseSerializer,
+    SIMPLE_CAMPAIGN_EXAMPLE,
+    SIMPLE_CAMPAIGN_DONATION_EXAMPLE,
```

Ruff (0.12.2)

`.serializers.SIMPLE_CAMPAIGN_DONATION_EXAMPLE` imported but unused

Remove unused import: `.serializers.SIMPLE_CAMPAIGN_DONATION_EXAMPLE`

(F401)